util/src/main/scala/org/ensime/util/Show.scala

```diff
@@ -0,0 +1,18 @@
+package org.ensime.util
+
+trait Show[T] {
```

Comment:
hehe, yeah maybe it's a bit much just for one typeclass (especially something so trivial). Can we perhaps do what we want without the `Show` typeclass? If we're going to use `Show`, we should be consistent and use it everywhere.